unlock-app/src/config.js

```diff
@@ -144,6 +144,26 @@ export default function configure(
     blockTime = 8000
   }
 
+  if (env === 'dev-kovan') {
+    // In dev-kovan, the network can only be Kovan
+    isRequiredNetwork = networkId => networkId === 42
+    chainExplorerUrlBuilders.etherScan = address =>
+      `https://kovan.etherscan.io/address/${address}`
+    requiredNetworkId = 42
+    paywallUrl = 'https://'
```

Comment:
Probably worth including in the wiki that right now the integration is _only_ functional for the dashboard, not the paywall.

Reply:
Yeah, good idea.